### ISSUE TITLE
Remove the  Routing option when registering a template from the UI 

### ISSUE THREAD

**Issue by kiranchavala** [apache/cloudstack#7774]
ISSUE TYPE

 * Improvement request

COMPONENT NAME

Component: UI 

CLOUDSTACK VERSION

Cloudstack 4.18

SUMMARY

Remove the  Routing option when registering a template from the UI 

Steps to reproduce the issue 

Register a template 

![routing-option](https://github.com/apache/cloudstack/assets/1401014/5c32328c-adfe-483a-8514-5bf90e8af00e)

Currently the routing option is only used by the Virtual router. 

If users need, they can update template type to ROUTING

![Update template](https://github.com/apache/cloudstack/assets/1401014/6e2778e5-87ce-4306-9b8a-4052879cc29a)

Since we have a streamlined approach with making a template a systemvmtemplate

https://www.shapeblue.com/systemvm-template-upgrade-improvements/

we can consider to remove the routing option when registering a template

**Comment by DaanHoogland** [comment]
@kiranchavala , I don't agree with this. Because the user can edit a template, shouldn't mean they cannot immediately on registering give it the right properties.

**Comment by kiranchavala** [comment]
@DaanHoogland  then we should also show other template types (user,system,builtin)  when registering a template.

why is it only limited it to only routing 

Also the documentation related to registering a template doesn't mention about the routing part

**Comment by weizhouapache** [comment]
@DaanHoogland @kiranchavala 
do we think this is a bug and need a fix ?

**Comment by DaanHoogland** [comment]
> @DaanHoogland @kiranchavala do we think this is a bug and need a fix ?

I don't think so, it is a polishing issue with low prio. It can be added in any release.
